### DISCUSSION TITLE
Emoji 15 Alpha

### DISCUSSION THREAD

**Discussion by C1710** [C1710/blobmoji#113] [General]
![Blobmoji 15 Alpha release picture](https://user-images.githubusercontent.com/10349490/164793614-4967abed-3057-48ff-91a2-571ecaf8b3ff.jpg)
This release will be continuously updated - it is by no means final (especially the tables used here, there has not been an official release yet)!

Note: Unicode 15 is not yet finalized, things can still change, there is no real software support yet.

In other news: I confused hyacinth with grape hyacinth. Although it is rather similar especially at emoji size, it will be changed/corrected later.

<hr /><em>This discussion was created from the release <a href='https://github.com/C1710/blobmoji/releases/tag/v15.0-alpha.1'>Emoji 15 Alpha</a>.</em>

**Comment by mistu01** [comment]
anymore update??!

**Reply by C1710** [reply]
I'm currently busy with other stuff, so I'm not sure when I have time for the final release.

However, I think it all works, with only the hyacinth emoji showing the (slightly) wrong plant.

**Reply by C1710** [reply]
I just changed hyacinth to the design from Noto Emoji and merged into the main/master branch 😅
